src/orderbook.js

```diff
@@ -164,7 +201,15 @@ OrderBook.EVENTS = [
   'offer_changed', 'offer_funds_changed'
 ];
 
-OrderBook.DEFAULT_TRANSFER_RATE = 1000000000;
+OrderBook.DEFAULT_TRANSFER_RATE = Amount.from_json(1000000000);
+
+OrderBook.NOTIFY_THROTTLE_TIME = 4000;
```

Comment:
And it is good idea to change `NOTIFY_THROTTLE_TIME` to something like 100 ms, but autobridge calculation still must be throttled with something big, because it is too slow, and we risk to get same situation like now - when autobridge calculation gets called more often than it can complete